terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java

```diff
@@ -38,8 +42,8 @@ public class ReservationReportPdfStamperView extends AbstractPdfStamperView {
 
     private static final float REFERENCE_NAME_VARIABLE_FONTSIZE = 8.0F;
 
-    private final SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd");
-
+    private DateTimeFormatter fmt = DateTimeFormatter.ofPattern("yyyy/MM/dd");
```

Comment:
- [ ] When using DateTimeFormatter to handle the western calendar, "uuuu" is more suitable than "yyyy".

Reply:
It was fixed as pointed out